workflows/flows/import_genomes_flow.py

```diff
@@ -72,7 +67,7 @@ def get_catalogue(options):
             "version": options["catalogue_version"],
             "name": f"{options['catalogue_name']} v{options['catalogue_version']}",
             "biome": biome,
-            "result_directory": options["catalogue_dir"],
+            "result_directory": f"/nfs/public/services/metagenomics/results/{options['results_directory']}",
```

Comment:
This one.. why does it have to be the full path here? Can we just provide the full path as the arg?

Reply:
since this path is basically constant, i.e /nfs/public/services/metagenomics/results/
I didn't think it necessary to be given as an argument. Since we always know what it is basically. 
However, I'll make the change to use it as an arg